docs/Changelog.md

```diff
@@ -45,7 +45,8 @@
     * The editor can be easily switched backed to 'raw' version similarly to the current Kafka Sink
   * [#6545](https://github.com/TouK/nussknacker/pull/6545) Non-physical columns are properly handled in Table Sink now
     * User need to provide only persisted column values now - values for computed columns and virtual columns are not required anymore
-* [#6559](https://github.com/TouK/nussknacker/pull/6559) [#6559](https://github.com/TouK/nussknacker/pull/6559) Extension: Move previousValue and delay components from unbounded to base Flink components
+* [#6559](https://github.com/TouK/nussknacker/pull/6559) Extension: Move previousValue and delay components from unbounded to base Flink components
+* [#6572](https://github.com/TouK/nussknacker/pull/6572) Improvement: Introduction to FlinkPeriodicDeploymentManagerProvider
```

Comment:
User or even admin that deploy this application won't know what does `FlinkPeriodicDeploymentManagerProvider` mean. We don't have any documentation around this, it is not described anywhere in a blog post. I think that we should skip this changelog entry. It is for our internal deployments purpose